dstep/translator/Record.d

```diff
@@ -34,32 +34,141 @@ void translatePackedAttribute(Output output, Context context, Cursor cursor)
         output.singleLine("align (1):");
 }
 
+struct BitField
+{
+    string type;
+    string field;
+    uint width;
+}
+
+BitField[] translateBitFields(
+    Context context,
+    Cursor[] cursors)
+{
+    void pad(ref BitField[] bitFields, uint totalWidth)
```

Comment:
This could perhaps be a static function.

Reply:
There's nothing wrong with it being nested but since it's not using anything from the surrounding context (like the parameters of `translateBitField`) it can be `static`. That will avoid allocating the extra context pointer but it will still be lexically nested in `translateBitFields`. It's just a minor detail but I think it keeps the code nice and tidy.

Reply:
I think any decent compiler should figure it out...

Reply:
My point if its nested its clear that its needed only in the scope of parent function (e.g. it doesn't pollute the global namespace), to make it static I would need to extract it and the coupling between the functions becomes less clear.

Reply:
To make it static, you just add the `static` keyword.

Reply:
Heh, the obvious solutions are hardest to see : ) I'll fix it on next PR...